src/utils/Sort.js

```diff
@@ -727,3 +727,16 @@ export function countGroup(group) {
   }
   return group.length;
 }
+
+export function sortForCSVDownload(cards) {
+  var exportCards = [];
+  cards = sortDeep(cards, 'Color', 'CMC');
+  for (const firstGroup of cards) {
+    for (const secondGroup of firstGroup[1]) {
```

Comment:
This doesn't look right... we are we doing `firstGroup[1]`?

Reply:
Each group produced by `sortDeep` is an array where the 0 index is the category (like `White` or `3`) and the 1 index is a subarray with either second groups or the cards themselves.

Reply:
Right, because that was used to write the headers. Looks good to me then